src/bp/ui-studio/src/web/components/ChatEmulator/Emulator.jsx

```diff
@@ -189,7 +189,7 @@ export default class EmulatorChat extends React.Component {
       },
       () => {
         this.textInputRef.current.focus()
-        if (callback) {
+        if (_.isFunction(callback)) {
```

Comment:
probably lodash is already imported but do you really need it ? `typeof callback === "function"`

Reply:
Only used because lodash was already there, but yeah I weighted in on both ways